src/examples/Navbars/DefaultNavbar/index.js

```diff
@@ -80,7 +80,7 @@ function DefaultNavbar({ transparent, light, action }) {
           <DefaultNavbarLink
             icon="account_circle"
             name="sign up"
```

Comment:
In the whole app using the sign-up/create-account but here it is authentication/sign-up/create can you modify either here or entire app please